.buildkite/pipeline.yml

```
@@ -12,4 +12,45 @@ steps:
     command: "make test"
     key: "tests"
     artifact_paths:
-      - "coverage.out"
\ No newline at end of file
+      - "coverage.out"
+
+  - label: ":sonarqube: Static Code Analysis"
+    env:
+      VAULT_SONAR_TOKEN_PATH: "kv/ci-shared/serverless/shared-analysis-token"
+    agents:
+      image: "docker.elastic.co/cloud-ci/sonarqube/buildkite-scanner:latest"
+    command:
+      - "buildkite-agent artifact download coverage.out ."
+      - "/scan-source-code.sh"
+    soft_fail: true
+    depends_on: "tests"
+
+  # Need go 1.20 in the serverless-docker-builder to enable this 
+  #- label: ":go: Build"
+  #  command: "make all"
+
+  # multiarch dockerx build
```

```suggestion
  # multi-arch container image build
```

I assume you wanted to say `docker buildx`. `drivah` doesn't use `dockerx build` but [`buildah`](https://github.com/elastic/drivah/blob/5a59c63927204b53d364b4c111de950f69e5b1ad/drivah/engine/buildah.py#L32), by default. It could use [`docker build`](https://github.com/elastic/drivah/blob/5a59c63927204b53d364b4c111de950f69e5b1ad/drivah/engine/docker.py#L41), but not `docker buildx build`.

:)